jupyter_collaboration/events/awareness.yaml

```diff
@@ -0,0 +1,43 @@
+"$id": https://schema.jupyter.org/jupyter_collaboration/awareness/v1
+"$schema": "http://json-schema.org/draft-07/schema"
+version: 1
+title: Collaborative awareness events
+personal-data: true
+description: |
+  Awareness events emitted from server-side during a collaborative session.
+type: object
+required:
+  - level
+  - room
+  - user
+  - action
+properties:
+  level:
+    enum:
+      - INFO
+      - DEBUG
+      - WARNING
+      - ERROR
+      - CRITICAL
```

Comment:
I don't think `level` is needed in this event, since this isn't really logging type event, but rather an action in the server.

Reply:
make sense, will address.